### ISSUE TITLE
SDL2: Music is sometimes too fast or too slow

### ISSUE THREAD

**Issue by mrtnptrs** [CorsixTH/CorsixTH#37]
The music is sometimes to slow, fast or it stutters or it sounds completely different as it should. 

This issue has so far only been verified on virtual machines..

**Comment by jorgenpt** [comment]
I've pinged one of the SDL_mixer authors about cutting a new release, since it's still at 2.0.0.

**Comment by mrtnptrs** [comment]
Then I will update the travis script as soon as the new version comes out :)

**Comment by TheCycoONE** [comment]
If they still haven't cut a new release approaching the time we want to cut our release I'll be including an hg version in the Windows release, because this issue has been fixed upstream for a long time now.  We should have enough time to test it in the release candidate phase to test it.

**Comment by jorgenpt** [comment]
I heard back -- they're planning on releasing a new version of the "satellite" libraries when they release SDL 2.0.4. Still waiting to hear what the schedule for 2.0.4 is.

**Comment by TheCycoONE** [comment]
Should be fixed in 0.50 beta 1 which uses an hg release of SDL2 mixer.

**Comment by bobcanthelpyou** [comment]
If it is the same issue it is fixed in the SDL2 mixer packages of Debian Jessie, too
https://packages.debian.org/jessie/libsdl2-mixer-2.0-0
```
libsdl2-mixer (2.0.0+dfsg1-5) unstable; urgency=medium

  * Add patch fix-750706-upstream-8ef083375857.patch backported from
    upstream to fix playing speed with timidity backend.  Thanks Sylvain
    Beucler (Closes: #750706)

 -- Manuel A. Fernandez Montecelo <mafm@debian.org>  Sat, 25 Oct 2014 11:07:28 +0100
```
http://metadata.ftp-master.debian.org/changelogs//main/libs/libsdl2-mixer/libsdl2-mixer_2.0.0+dfsg1-5_changelog

But it could be still a problem on Ubuntu Trusty 
http://packages.ubuntu.com/trusty/libsdl2-mixer-2.0-0

**Comment by TheCycoONE** [comment]
In any case, it's not our issue, and our official builds will not include the broken version.  Closing.
